### DISCUSSION TITLE
Buyer App Commission and Finder/Fulfilment Fees

### DISCUSSION THREAD

**Discussion by nebi004** [ONDC-Official/ONDC-FIS-Specifications#35] [Q&A]
1) Is there any difference between buyer app commission and finder/fulfilment fees for lending products? Or Buyer app will charge only 1 thing i.e. Commission which will include finder and fulfilment fees also?

2) Will there be any standard commission % defined by ONDC?

3) How will lender make payments of these commission to buyer apps? Also when the number buyer apps will increase, how will commission payments be managed?

**Comment by 92shreyansh** [comment]
@Antriksh-Parmar

**Reply by Antriksh-Parmar** [reply]
@nebi004 
1. No difference. 
2. No. It's up to the buyer app to determine the BFF.
3. It'll be done off the network initially, we'll push for daily settlements.
